src/tactic/model_converter.cpp

```diff
@@ -76,6 +82,24 @@ class concat_star_model_converter : public concat_star_converter<model_converter
         }
         UNREACHABLE();
     }
+
+    virtual void operator()(svector<symbol> & r, unsigned goal_idx) {
+        unsigned num = this->m_c2s.size();
+        for (unsigned i = 0; i < num; i++) {
+            if (goal_idx < this->m_szs[i]) {
```

Comment:
how are m_szs and m_c2s.size related? Hopefully num == m_szs.size(), but if so, then why not use num = m_szs.size()?